server/src/main/java/io/crate/protocols/postgres/PGError.java

```diff
@@ -74,6 +76,21 @@ public static PGError fromThrowable(Throwable throwable) {
         if (throwable instanceof IllegalArgumentException || throwable instanceof UnsupportedOperationException) {
             status = PGErrorStatus.FEATURE_NOT_SUPPORTED;
         }
-        return new PGError(status, SQLExceptions.messageOf(throwable), throwable);
+        String message = SQLExceptions.messageOf(throwable);
+        if (message == null) {
```

Comment:
`message` will never be `null` here because of how `messageOf` is implemented. 

Maybe it would be an option to integrate the strack-trace element logic into `messageOf`?, not sure what else would be affected by that.

Reply:
What do you think about the idea to only add the stacktrace based on the `error_trace` flag similiar we do at the http client ? https://github.com/crate/crate/blob/eedbca041e1781ceebc52ce61d93f0975414660e/server/src/main/java/io/crate/rest/action/SqlHttpHandler.java#L169

Reply:
How would a PostgreSQL client set that flag? 

The logic there wasn't meant to include the full stacktrace, but to have an error message that is a bit more meaningful. For example `NullPointerException` often don't have a message.

For including the stacktrace we should actually additionally look into adding the `W` parts from [here](https://www.postgresql.org/docs/9.2/protocol-error-fields.html)

Reply:
There is also stacktrace handling before the error is sent to the client. So what do you think, we could also just remove the stacktrace code in both client then, since it is handled already ?
https://github.com/crate/crate/blob/ab23d167ab2651bd65ce260a7964543116a47a19/server/src/main/java/io/crate/protocols/postgres/Messages.java#L190

Reply:
> How would a PostgreSQL client set that flag?

Oh, my assumption was it was a server side parameter.

> The logic there wasn't meant to include the full stacktrace, but to have an error message that is a bit more meaningful. For example `NullPointerException` often don't have a message.

Ok, I will try to come up with something meaningful.